build/config.js

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) 2013 - present UTN-LIS
```

Comment:
@emravera Note that I'm using a shorter license header to include in all of the files.